scripts/pkg_in_pipe/README.md

```diff
@@ -0,0 +1,14 @@
+# Packages in pipe report generator
+
+Generates an html report with the packages in the current tags.
+
+# Requirements
+
+An environmnt variable `PLANE_TOKEN` must be defined when running the generator.
+
+# Run in docker
+
+```sh
+docker build -f pkg_in_pipe .
```

Comment:
```suggestion
docker buildx build  . -t pkg_in_pipe
```

`docker buildx` replaces `docker build` see the "Important" note here https://docs.docker.com/reference/cli/docker/build-legacy/

Reply:
`docker build` is an alias to `docker buildx build` and buildkit is the default build engine since docker 23, so I we should be good with `docker build` :-)

ref:
* https://docs.docker.com/reference/cli/docker/buildx/build/
* https://docs.docker.com/build/buildkit/

Reply:
good catch for the `-t`!